website/docs/docs/collaborate/explore-projects.md

```diff
@@ -15,7 +15,17 @@ With dbt Explorer, you can view your project's [resources](/docs/build/projects)
 - You have at least one successful job run in the deployment environment. Note that [CI jobs](/docs/deploy/ci-jobs) do not update dbt Explorer.
 - You are on the dbt Explorer page. To do this, select **Explore** from the navigation in dbt Cloud.
 
-<Lightbox src="/img/docs/dbt-cloud/explore-nav.jpg" width="95%" title="Access dbt Explorer from dbt Cloud by clicking Explore in the navigation."/>
+## Overview page
+
+Navigate the dbt Explorer overview page to access your project's resources and metadata. The page includes the following sections:
+
+- **Search bar** &mdash; [Search](#search-resources) for resources in your project by keyword. You can also use filters to refine your search results.
+- **Sidebar** &mdash; Use the left sidebar to access model [performance](/docs/collaborate/model-performance), [project recommendations](/docs/collaborate/project-recommendations) in the **Project details** section. Browse your project's [resources, file tree, and database](#browse-with-the-sidebar) in the lower section of the sidebar.
+- **Lineage graph** &mdash; Explore your project's or account's [lineage graph](#project-lineage) to visualize the relationships between resources.
+- **Latest updates** &mdash; View the latest changes or issues related to your project's resources, including the most recent job runs, changed properties, lineage, issues, and more.
```

Comment:
Small suggestion so folks aren't left wondering at the end of the sentence since we hit the big blocks there
```suggestion
- **Latest updates** &mdash; View the latest changes or issues related to your project's resources, including the most recent job runs, changed properties, lineage, and issues.
```